### DISCUSSION TITLE
Beets-audible: A Beets Plugin That Makes Audiobook Collections Compatible With Audiobookshelf

### DISCUSSION THREAD

**Discussion by Neurrone** [advplyr/audiobookshelf#411] [Show and tell]
I've developed a [Beets Audible plugin](https://github.com/Neurrone/beets-audible) that implements the process described in [seanap's audiobook guide](https://github.com/seanap/Plex-Audiobook-Guide). It automates most of the work needed to make audiobooks compatible with Audiobookshelf. It is also cross-platform and much faster and easier to use than Mp3Tag.

I'd consider it beta quality. I've used it for a few weeks now. It works for the vast majority of books that I've tried, with some limitations documented in the readme.

Hope someone finds this useful. My hope is for this to become the new gold standard of audiobook management.

**Comment by Jdiesel87** [comment]
Cool project. It's been a while since I've used Beets. Is there anything preventing the Chromaprint/Acoustid Plugin from being used with audiobooks?

**Reply by Neurrone** [reply]
I believe that plugin is only useful if there is metadata for that fingerprint to match to.

Given the wide variety of ways that an audiobook can be saved (single file, chapterized, split across files but not according to chapterized), that throws off any matches even if such metadata existed somewhere centrally that can be used for tagging.
